Week3/homework/step4.js

```diff
@@ -1,8 +1,14 @@
 'use strict';
 
 // Add your code here
+function createBase(firstNum) {
+
+    return function add(secondNum) {
+        return firstNum + secondNum;
+    };
+}
 
 const addSix = createBase(6);
 
-addSix(10); // returns 16
-addSix(21); // returns 27
+console.log(addSix(10)); // returns 16
+console.log(addSix(21)); // returns 27
```

Comment:
✅ Excellent! Nice naming of the variables here.  Makes it very clear what is happening.